azure-iot-hub-devicesdk/azure/iot/hub/devicesdk/sync_clients.py

```diff
@@ -192,16 +192,6 @@ def callback():
 
         enable_complete.wait()
 
-    def _emit_connection_status(self):
-        """
-        The connection status is emitted whenever the client on the module gets connected or disconnected.
-        """
-        logger.info("emit_connection_status: {}".format(self.state))
-        if self.on_connection_state:
-            self.on_connection_state(self.state)
-        else:
-            logger.warn("No callback defined for sending state")
```

Comment:
this doesn't seem to be related to just moving tests. + there are still calls to this API from other parts of the file

Reply:
It's a duplicate. This functionality is provided elsewhere, and got duplicated during a merge

---
In reply to: [250847278](https://github.com/Azure/azure-iot-sdk-python-preview/pull/43#discussion_r250847278) [](ancestors = 250847278)